src/DeviceManagement.Administration/DeviceManagement.Administration/readme.md

```diff
@@ -38,6 +38,11 @@ subject-prefix: ''
 
 ``` yaml
 directive:
+# Pluralize
+  - where:
+      subject: (.*)OnPremisConnection(.*)
+    set:
+      subject: $1OnPremisesConnection$2
```

Comment:
We should run this through the workload team to get their feedback while considering [this PowerShell naming rule](https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/strongly-encouraged-development-guidelines?view=powershell-7.1#use-a-specific-noun-for-a-cmdlet-name-sd01).
> To enhance the user experience, the noun that you choose for a cmdlet name should be singular. For example, use the name Get-Process instead of Get-Processes. It is best to follow this rule for all cmdlet names, even when a cmdlet is likely to act upon more than one item.

Also, since we already have commands that use `OnPremise` (singular form) in their names, I'd say let's stick to the singular form (`OnPremise`) for consistency with already published commands, e.g., `Get-MgOnPremisePublishingProfile`. If we have to rename them to `OnPremises`, let's do it for all commands and alias them back to their `OnPremis` to avoid a breaking change.

Reply:
Hi @peombwa @georgend , "OnPremisesConnection" is the entity name in our CloudPC workload, and the singular non is just "OnPremisesConnection", could you don't change it to "OnPremisConnection". You can see this is our [list OnPremisesConnections API ](https://docs.microsoft.com/en-us/graph/api/virtualendpoint-list-onpremisesconnections?view=graph-rest-beta&tabs=http) and [get one OnPremisesConnection API](https://docs.microsoft.com/en-us/graph/api/cloudpconpremisesconnection-get?view=graph-rest-beta&tabs=http)